src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -540,18 +540,9 @@ private function isMaxDepthReached(array $attributesMetadata, string $class, str
      * We must not mix up the attribute cache between parent and children.
      *
      * {@inheritdoc}
-     *
-     * @param string|null $format
      */
-    protected function createChildContext(array $parentContext, $attribute/*, ?string $format */)
+    protected function createChildContext(array $parentContext, string $attribute, ?string $format): array
```

Comment:
should have null as default, otherwise that's a hard BC break

Reply:
The method is internal. That should be fine, right?

Reply:
`@internal` is not transitive
once all are correctly annotated on 4.4, sure :)

Reply:
fixed in 2b8e44164e